test/utils/filterDuplicates.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from 'chai';
+import filterDuplicates from '../../src/utils/filterDuplicates';
+
+describe('filterDuplicates', () => {
+	it('no duplicates - not modified', () => {
+		const input = [1, 2, 3, 4];
+		const output = input.filter(filterDuplicates);
+		expect(output).deep.equal(input);
+	});
+	it('duplicates - removes duplicates', () => {
+		const input = [1, 1, 2, 3, 2, 3, 4, 4, 4, 4, 4];
```

Comment:
🤔 surely this works great, but the example looks a bit reductionist. no?

Reply:
Open to other test case suggestions. The idea here is that this covers elements that are in mixed order and when there is more than 1 to remove.